src/components/vocabularies/IdentityCheckCredential.ts

```diff
@@ -36,7 +36,7 @@ export type IdentityCheckCredentialType =
   | "IdentityCheckCredential"
   | "VerifiableCredential";
 
-export interface Credentials {
+export interface Credential {
```

Comment:
Ah interesting, yeah that makes more sense.
FYI we'd this means we're diverging from the JSON Schema at https://vocab.london.cloudapps.digital/

Probably OK as if we tried to regenerate these we'd lose the refactor of common elements out anyway.

Reply:
We talked about this and decided it was ok - the published vocab is in need of a bit of love anyway so diverging from it is fine.